common/djangoapps/enrollment/views.py

```diff
@@ -612,60 +633,87 @@ def post(self, request):
                 }
                 consent_client.provide_consent(**kwargs)
 
-            enrollment_attributes = request.data.get('enrollment_attributes')
```

Comment:
This section was not removed.  I changed the indentation and shifted it down a bit.